docs/upgrade_1-2_to_1-3.md

```diff
@@ -36,21 +36,44 @@
 +      uploaderType: restic
 ```
 
-> **Note:** OADP will be favoring Kopia over Restic in the near future.
+> **Note:** In the next version of OADP the Restic option will be deprecated and Kopia will be come the default uploaderType.
 
 ## Upgrade steps
 
-### Create new backup
+### If the OADP 1.2 tech-preview DataMover feature is in use, please read the following.
 
-If you are using DataMover, you need to create new backups before upgrade, because the 1.2 DataMover backups will not work after upgrade. We suggest to create Restic backups.
+OADP 1.2 DataMover backups can NOT be restored with OADP 1.3. To prevent a gap in the data protection of your applications we recommend the following.
 
-### Copy old DPA
+* If on cluster backups are sufficient and CSI storage is available
+  * Backup the applications with a CSI backup
+
+* If off cluster backups are required
+  * Backup the applications with a filesystem backup using the `--default-volumes-to-fs-backup=true` option.
```

Comment:
Since this will be in 1.2, we can say its restic, right?

would also tell about CLI x UI

"Backup the applications with restic filesystem backup using the `--default-volumes-to-fs-backup=true` flag, if using velero CLI, or using `defaultVolumesToFsBackup` field in Backup CR."

Reply:
agree.. thanks